frame/support/src/lib.rs

```diff
@@ -1388,7 +1388,7 @@ pub mod pallet_prelude {
 /// ### Macro expansion
 ///
 /// For each storage item the macro generates a struct named
-/// `_GeneratedPrefixForStorage$NameOfStorage`, implements `StorageInstance` on it using the 
+/// `_GeneratedPrefixForStorage$NameOfStorage`, implements `StorageInstance` on it using the
```

Comment:
```suggestion
/// `_GeneratedPrefixForStorage$NameOfStorage`, and implements [`StorageInstance`] on it using the
```

Reply:
Not sure `StorageInstance` will resolve as a doc link. 

I don't think I intended to edit this line at all

Reply:
Yes it is in path https://crates.parity.io/frame_support/traits/trait.StorageInstance.html
So it might be needed to write is `traits::StorageInstance` instead

Reply:
I've started to add ```[`Thing`]``` sort of blindly in the hope that it all links up in the end; if it works that's great, if it doesn't it's not a big deal. But yeah, I should probably check if the paths are in scope.